docs/src/main/asciidoc/mongodb-panache-guide.adoc

```diff
@@ -0,0 +1,525 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Simplified MongoDB with Panache
+
+include::./attributes.adoc[]
+:config-file: application.properties
+
+MongoDB is a well known NoSQL Database that is widely used, but using its raw API can be cumbersome as you need to express your entities and your queries as a MongoDB link:https://mongodb.github.io/mongo-java-driver/3.11/bson/documents/#document[`Document`].
+
+MongoDB with Panache provides active record style entities (and repositories) like you have in link:hibernate-orm-panache-guide.html[Hibernate ORM with Panache] and focuses on making your entities trivial and fun to write in Quarkus.
+
+It is build on top of the link:mongo-guide.html[MongoDB Client] extension.
+
+WARNING: This extension is still experimental, feedback are welcome on the [mailing list](https://groups.google.com/d/forum/quarkus-dev) or as issue in our [github issues tracker](https://github.com/quarkusio/quarkus/issues).
```

Comment:
```suggestion
WARNING: This extension is still experimental, feedback is welcome on the https://groups.google.com/d/forum/quarkus-dev[mailing list] or as issues in our https://github.com/quarkusio/quarkus/issues[GitHub issue tracker].
```
be careful, you used Markdown links. Please generate the output and check the formatting.